inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/postgres-cdc/src/main/java/org/apache/inlong/sort/postgre/PostgresValueValidator.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.postgre;
+
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.types.RowKind;
+
+/** The {@link RowDataDebeziumDeserializeSchema.ValueValidator} for Postgres connector.
+ *
+ * Copy from com.ververica:flink-connector-postgres-cdc-2.3.0
+ * */
+public final class PostgresValueValidator
```

Comment:
Any differences with the original class in flink-connector?

Reply:
PostgresValueValidator implements RowDataDebeziumDeserializeSchema.ValueValidator, but I modify RowDataDebeziumDeserializeSchema, so, setValueValidator(ValueValidator validator) need validator which implements modified RowDataDebeziumDeserializeSchema.ValueValidator. ~